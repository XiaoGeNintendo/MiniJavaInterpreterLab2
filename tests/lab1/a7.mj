int main(){
int result = 10;boolean a = false;boolean b = true;boolean sce = a and (b or (result++ == 10));
println(result);
println(a);
println(b);
println(sce);
return 0;
}