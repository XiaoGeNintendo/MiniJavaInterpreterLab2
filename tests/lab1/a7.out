10
false
true
false
Process exits with the code 0.